src/samplers/inversioncdt.jl

```diff
@@ -0,0 +1,30 @@
+struct IdentityPermute; end
```

Comment:
Could you add a few docstrings over the newly defined types? (Even when non-exported)